### PR TITLE
Array shift

### PR DESCRIPTION
**Pull request by nickmagruder**

### CODE REVIEW

**Comment by nickmagruder** [comment]
Will resubmit, without using array.splice.

 - [x] Top-level README “Table of Contents” is updated
 - [ ] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [ ] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

File: challenges/arrayShift/array-shift.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+'use strict'
+
+function insertShiftArray(arr, num) {
+    let arrayCount = 0;
+    for (let i = 0; i < arr.length; i++) {
+        arrayCount++;
+    }
+let index = Math.ceil(arrayCount / 2);
+arr.splice(index, 0, num);
```
Comment (adrianhuebner) [inline challenges/arrayShift/array-shift.js:9]: I would like to see this done in a way that isn't using .splice(
